### DISCUSSION TITLE
Chào mọi người mình mới toanh! Và cũng không hiểu tiếng Anh

### DISCUSSION THREAD

**Discussion by Niemdamme** [community/community#143606] [New to GitHub]
[[Mình]()](url) mới bước vào môi trường mới này mình vẫn còn nhiều thứ lạ lẫm. Mình chân thành cảm ơn tất cả các bạn làm ra nơi này,  và vấn đề đầu tiên của mình là : đầu tiên sản phẩm chưa công bố nhưng cũng sắp vậy khi mình mượn 1 ít tài nguyên chỉ là nguồn mở ở đây thì sẽ tính thế nằm không có nguồn tài nguyên bảo mật tuy chưa xong nhưng mình báo trước thẳng thắng dễ hơn mập mờ! 🤣 🤣 🤣 Dụng cụ nhỏ thôi nhưng lý tưởng cao đẹp! 😅😅😅😅! Thân chào mọi người!

- Tái bút : mình dốt tiếng anh, không biết làm vậy có đúng không, mong mọi người thông cảm Link này là bản kế hoạch có hứng thú thì mời ghé xem nhé! @Bestra @espressif @matterbridge @dockerhub ****https://github.com/Niemdamme/Escoyhome-smart-/blob/d935d9f6681377d0efadf1df11b2782a221fb631/README.md

**Comment by ghostinhershell** [comment]
Hi @Niemdamme, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
